app/assets/javascripts/models/data/project.js.coffee

```diff
@@ -87,9 +87,12 @@ ns.Project = do (ko,
       options = @serviceOptions.serialize()
       $(document).trigger('dataaccessevent', [@collection.id, options])
 
+      form_data = []
+      form_data.push method.form for method in @granuleAccessOptions().methods
       id: @collection.id
       params: param(@collection.granuleDatasource()?.toQueryParams() ? @collection.query.globalParams())
       serviceOptions: options
+      form: form_data
```

Comment:
This addition seems to be adding all of the forms, in their entirety to the saved data in a project, which is potentially quite large (AST_L1A). Can we pare it down to just the digest. And, really, just the selected digest?